src/components/[guild]/RolePlatforms/components/AddRoleRewardModal/components/SelectExistingPlatform.tsx

```diff
@@ -36,7 +30,7 @@ const SelectExistingPlatform = ({ onClose, onSelect }: Props) => {
     name: "rolePlatforms",
   })
 
-  const roleVisibility: Visibility = useWatch({ name: ".visibility" })
+  const roleVisibility = useWatch({ name: ".visibility" })
```

Comment:
Should we remove the leading `.` here as well?

Reply:
Double checked and it works correctly without the dot, so removed it, thx for pointing it out!